java/libs.javacapi/nbproject/org-netbeans-libs-javacapi.sig

```diff
@@ -301,6 +301,11 @@ intf com.sun.source.tree.ExpressionTree
 meth public abstract com.sun.source.tree.ExpressionTree getLeftOperand()
 meth public abstract com.sun.source.tree.ExpressionTree getRightOperand()
 
+CLSS public abstract interface com.sun.source.tree.BindingPatternTree
+intf com.sun.source.tree.PatternTree
```

Comment:
`PatternTree` - that must be related to upgrading to JDK14 syntax, right @jlahoda?